src/Containers/Search/SearchResult.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import { Col } from 'react-bootstrap';
+import ellipsize from 'ellipsize';
+import dateFormat from 'dateformat';
+import 'bootstrap/dist/css/bootstrap.css';
+import NavBar from '../../Components/NavBar/NavBar';
+import NotFound from './NotFound';
+import ResultCard from './ResultCard';
+import './ResultCard.scss';
+
+/**
+ * render the search result component
+ *
+ * @param {object} props
+ * @returns {object} jsx
+ */
+export const SearchResult = props => {
+  const { searchFilter } = props;
+  const { articles } = searchFilter;
+
+  return (
+    <React.Fragment>
+      <NavBar {...props} />
+      {articles.length === 0 ? (
+        <NotFound />
+      ) : (
+        <div className="d-flex flex-column results">
+          <Col md={12} sm={12} lg={8} className="result-column">
+            {articles.map(item => {
```

Comment:
@coolbeatz71 Destructuring is preferred here, to different item fields.

Reply:
Yes @gracelungu you are right